.travis.yml

```diff
@@ -8,11 +8,12 @@ env:
  - GHCVER=7.8.4 CABALVER=1.18
  - GHCVER=7.10.3 CABALVER=1.22
  - GHCVER=8.0.1 CABALVER=1.24
```

Comment:
while at it, do you mind updating that to GHC=8.0.2 as well ? :-)

Reply:
Done :)